jq.c

```diff
@@ -1275,6 +1275,52 @@ jq_exception_clear()
 	return;
 }
 
+/*
+ * Helper Method from JNIHelp.c from android source code
+ */
+static jmethodID FindMethod(JNIEnv* env,
```

Comment:
Could you naming the function following the naming rule? For example, jq_find_method